src/lib/api/feed-manip.ts

```diff
@@ -89,23 +92,33 @@ export class FeedViewPostsSlice {
       this.isOrphan = true
       return
     }
+    const root = reply.root
+    const rootIsView =
```

Comment:
What does `rootIsView` mean? I thought maybe it's about it being `PostView` but the other condition is `BlockedPost`. Is this about detecting the ones that have a string `uri` property? I think `NotFound` would have it too but it's not in the condition. So I think I'm missing what this boolean means and why it's in the condition below.

Reply:
Ah, yeah it's exactly that. Added check for NotFoundPost too, good call 👍